docs/monoDrive_home/getting_started/Windows.md

```diff
@@ -25,6 +25,12 @@
 
     Note that the editor may take a long time to compiling shaders, and may appear to get stuck at 95% or 99%. This is expected and you will need to wait for the compiling to finish. If no error is reported then the system is still compiling shaders, this only happens the first time opening. 
 
+    Note that the Simulator needs access to communicate to Public networks, select "Allow Access." If this window does not pop-up and there is a license error, see the [solution here](link)
```

Comment:
Should the period go after the quotes? `"Allow Access". `

Reply:
Yeah, this one is a weird one, good catch. Normally periods go inside quotations at the end of a sentence, but in this case, it gets in the way of technical writing. Thanks!